### PR TITLE
MULE-11120: added flow name and correlation id to the default logger

### PR DESCRIPTION
**Pull request by facunlk**

### CODE REVIEW

File: core/src/main/java/org/mule/runtime/core/api/Event.java [hunk 1]
```diff
@@ -502,6 +503,10 @@ static Event getCurrentEvent() {
    */
   static void setCurrentEvent(Event event) {
     CurrentEventHolder.currentEvent.set(event);
+    if (event != null) {
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/api/Event.java]: clear the values if the event is null

File: core/src/main/java/org/mule/runtime/core/api/Event.java [hunk 2]
```diff
@@ -502,6 +503,10 @@ static Event getCurrentEvent() {
...
+      ThreadContext.put("correlationId", event.getCorrelationId());
+      ThreadContext.put("originatingFlowName", event.getContext().getOriginatingFlowName());
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/api/Event.java]: this is no longer available. get the flowName form the component location in the context

File: core/src/main/java/org/mule/runtime/core/processor/AbstractRequestResponseMessageProcessor.java [hunk 3]
```diff
@@ -93,6 +93,7 @@ protected Event processRequest(Event request) throws MuleException {
   protected ReactiveProcessor processRequest() {
     return stream -> from(stream).map(event -> {
       try {
+        Event.setCurrentEvent(event);
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/processor/AbstractRequestResponseMessageProcessor.java]: @dfeist any opinion on this?

File: modules/launcher/src/main/java/org/mule/runtime/module/launcher/log4j2/LoggerContextConfigurer.java [hunk 4]
```diff
@@ -67,7 +67,7 @@
 
   private static final String MULE_APP_LOG_FILE_TEMPLATE = "mule-app-%s.log";
   private static final String MULE_DOMAIN_LOG_FILE_TEMPLATE = "mule-domain-%s.log";
-  private static final String PATTERN_LAYOUT = "%-5p %d [%t] %c: %m%n";
+  private static final String PATTERN_LAYOUT = "%-5p %d [%t] [%X{originatingFlowName} - %X{correlationId}] %c: %m%n";
```
Comment (elrodro83) [inline modules/launcher/src/main/java/org/mule/runtime/module/launcher/log4j2/LoggerContextConfigurer.java]: also modify the patterns in the log4j config files

**Comment by aiannucci** [comment]
Inactive and outdated, please open updated PR with mentioned changes

**Pull request closed without merging**